integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -57,6 +57,26 @@ void testCreateAndLoadSchema() {
     Assertions.assertEquals("((ID,001))", properties);
   }
 
+  @Test
+  void testAlterSchema() {
+    String testDatabaseName = "t_alter";
+    sql("create database " + testDatabaseName);
+    Assertions.assertTrue(
+        StringUtils.isBlank(getDatabaseMetadata(testDatabaseName).get("Properties")));
+
+    sql(String.format("alter database %s set dbproperties ('ID'='001')", testDatabaseName));
+    Assertions.assertEquals("((ID,001))", getDatabaseMetadata(testDatabaseName).get("Properties"));
+
+    sql(String.format("alter database %s set location '/user'", testDatabaseName));
```

Comment:
seems hive meta store doesn't support alter database location, the IT pass because the default database location is like 'hdfs://localhost:9000/user/hive/warehouse/t_alter.db' , could you remove the location test and add some description about why not test alter location.    location test in `testCreateAndLoadSchema` has the same problem could you fix it together?